pinot-common/src/main/java/com/linkedin/pinot/common/restlet/swagger/SwaggerResource.java

```diff
@@ -197,6 +227,93 @@ private void generateSwaggerForFinder(JSONObject pathObject, String routePath, F
     }
   }
 
+  private JSONObject schemaForType(Class<?> type) {
+    try {
+      JSONObject schema = new JSONObject();
+
+      schema.put("type", "object");
```

Comment:
What do you think would be a good name for this constant? All of those strings are from the Swagger spec and not reused anywhere else in the Pinot codebase.